Website/src/rendering/src/assets/css/components/SessionsGrid.css

```diff
@@ -8,21 +8,27 @@
     @apply xl:grid-cols-4;
 
     .grid-item {
-      @apply bg-white mx-auto w-full;
+      @apply bg-white mx-auto w-full relative;
 
       .image-hover-zoom {
         @apply overflow-hidden bg-cover bg-center;
       }
 
-      img {
-        @apply transform scale-100 duration-300 ease-in-out;
-      }
+      transform: scale(1);
```

Comment:
This transform and transition should be after the `.grid-item` first `@apply`. Not after another class definition.